Solutions/Microsoft 365 Defender/Workbooks/MDO-Workbook.json

```diff
@@ -0,0 +1,1317 @@
+                      "maximumSignificantDigits": 3,
+                      "maximumFractionDigits": 2
+                    }
+                  }
+                },
+                "showBorder": false
+              }
+            },
+            "customWidth": "30",
+            "conditionalVisibility": {
+              "parameterName": "Nav",
+              "comparison": "isNotEqualTo",
+              "value": "Authentication"
+            },
+            "name": "query - 5 - Copy - Copy"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "EmailEvents\r\n| summarize count()by DeliveryLocation",
+              "size": 0,
+              "title": "Original Email Delivery Location count",
+              "timeContextFromParameter": "TimeRange",
+              "exportedParameters": [
+                {
+                  "fieldName": "SenderMailFromAddress",
+                  "parameterName": "Sender2",
+                  "defaultValue": "All"
+                },
+                {
+                  "fieldName": "NetworkMessageIds",
+                  "parameterName": "NetworkMessageIdsSender",
+                  "parameterType": 1,
+                  "defaultValue": "[]"
+                }
+              ],
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "crossComponentResources": [
+                "{Workspace}"
+              ],
+              "gridSettings": {
+                "formatters": [
+                  {
+                    "columnMatch": "SenderMailFromAddress",
+                    "formatter": 0,
+                    "formatOptions": {
+                      "customColumnWidthSetting": "29ch"
+                    }
+                  },
+                  {
+                    "columnMatch": "DistinctMessages",
+                    "formatter": 4,
+                    "formatOptions": {
+                      "palette": "blue",
+                      "customColumnWidthSetting": "20ch"
+                    }
+                  },
+                  {
+                    "columnMatch": "DistinctRecipients",
+                    "formatter": 4,
+                    "formatOptions": {
+                      "palette": "blue",
+                      "customColumnWidthSetting": "21ch"
+                    }
+                  },
+                  {
+                    "columnMatch": "DistinctRecipientDomains",
+                    "formatter": 4,
+                    "formatOptions": {
+                      "palette": "blue",
+                      "customColumnWidthSetting": "27ch"
+                    }
+                  },
+                  {
+                    "columnMatch": "NetworkMessageIds",
+                    "formatter": 5
+                  }
+                ]
+              }
+            },
+            "customWidth": "35",
```

Comment:
Can we change this to 50 and make the adjacent query also of the same width.

Reply:
done

Reply:
Not updated